scipy/ndimage/_support_alternative_backends.py

```diff
@@ -0,0 +1,72 @@
+import functools
+from scipy._lib._array_api import (
+    is_cupy, is_jax, scipy_namespace_for, SCIPY_ARRAY_API
+)
+
+import numpy as np
+from ._ndimage_api import *   # noqa: F403
+from . import _ndimage_api
+from . import _dispatchers
+__all__ = _ndimage_api.__all__
+
+
+MODULE_NAME = 'ndimage'
+
+
+def dispatch_xp(dispatcher, module_name):
+    def inner(func):
+        @functools.wraps(func)
+        def wrapper(*args, **kwds):
+            xp = dispatcher(*args, **kwds)
+
+            # try delegating to a cupyx/jax namesake
```

Comment:
```suggestion
            # try delegating to a cupyx/jax namespace
```

Reply:
Why? It does not delegate to a _namespace_, it delegates to an identically-named function from a namespace.

Reply:
okay :)